exercises/typescript/08-classes/01-problem/src/classes/hello-dino.test.ts

```diff
@@ -0,0 +1,15 @@
+import DinoKeeper from "./hello-dino";
+import { strict as assert } from "assert";
+import "mocha";
```

Comment:
We should not need `mocha` for anything (we are using Node’s built-in test runner).